src/ethereum/osaka/fork.py

```diff
@@ -705,6 +709,13 @@ def apply_body(
     for i, tx in enumerate(map(decode_transaction, transactions)):
         process_transaction(block_env, block_output, tx, Uint(i))
 
+    validate_inclusion_list(
```

Comment:
I want to share an idea about an edge case testing. It would be nice if we could test such situations that some withdrawal address become available to cover some IL transaction that they couldn't. That missing IL transaction should not reorg the block.